apps/finance/contracts/Finance.sol

```diff
@@ -674,14 +667,13 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         transaction.token = _token;
         transaction.entity = _entity;
         transaction.date = getTimestamp64();
-        transaction.reference = _reference;
```

Comment:
@bingen It might be nice to re-order this to reflect the order of the the [`Transaction`](https://github.com/aragon/aragon-apps/blob/03cb159a3f1280c9f7d076433aad8c0b533b6659/apps/finance/contracts/Finance.sol#L67) struct itself. Right now the storage is being set out of order, and I wouldn't count on the compiler to optimize it correctly for us.